app/javascript/react/screens/App/Overview/screens/MappingWizard/components/DualPaneMapper/DualPaneMapperCount.js

```diff
@@ -1,9 +1,10 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-const DualPaneMapperCount = ({ selectedItems, totalItems }) => (
-  <div className="dual-pane-mapper-count">{sprintf(__('%s of %s items selected'), selectedItems, totalItems)}</div>
-);
+const DualPaneMapperCount = ({ selectedItems, totalItems }) => {
+  const message = totalItems === 1 ? __('%s of %s item selected') : __('%s of %s items selected');
+  return <div className="dual-pane-mapper-count">{sprintf(message, selectedItems, totalItems)}</div>;
```

Comment:
Is this use of `sprintf` ok @AparnaKarve?

Reply:
`sprintf` looks good since all the gettext complexity is outside of `sprintf`, in `message`